node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -181,6 +189,15 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			collation_request_duration: prometheus::register(
```

Comment:
the default buckets may need some adjustment

Reply:
The default buckets are `0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0` seconds - should that be adjusted to somehow more long intervals or vice versa?

Reply:
I feel like we'd want more info within the 100ms to 1s range especially.